src/NLog.Targets.Syslog/Policies/Throttling.cs

```diff
@@ -61,7 +61,10 @@ private void ApplyDeferment(int waitingLogEntries)
 
             var deferment = FixedTime(Delay, waitingLogEntries);
             InternalLogger.Warn("Applying defer throttling strategy ({0} ms)", deferment);
-            Thread.SpinWait(deferment);
+            if (deferment <= 0)
+                Thread.Sleep(1);
+            else
+                Thread.Sleep(deferment);
```

Comment:
Well the idea is to wait a little before trying again. Not waiting at all is too aggressive. And Sleep(1) is not very long time.